landing/metadata.json

```diff
@@ -0,0 +1,53 @@
+{
+  "version": "0.0.1",
+  "categoriesTiles": {
+    "introText": "What would you like to do today?",
+    "tiles": [
+      {
+        "title": "Deploy a Blockchain Node",
+        "description": "Easy and low cost hosting for your blockchain nodes (RPC servers, Validators and more)",
+        "buttonText": "Choose a Template",
+        "route": "/landing/node-deployment",
+        "icon": "xrayView",
```

Comment:
file is named xray-view https://github.com/akash-network/deploy-templates/tree/main/template-icons/landing/icons - seems like either this file or the svg filename need to change for this to work

Reply:
Hey Anil, i'll add the paths as well, but this is how i've referenced the SVG's for now.

https://github.com/kaushik-rishi/console/blob/feat-new-landing-page/web/src/components/Icons/index.tsx#L545-L562

https://github.com/kaushik-rishi/console/blob/feat-new-landing-page/web/src/components/Icons/index.tsx#L594-L598

Reply:
Added the svg file paths as well

Reply:
oh, I didn't relize they were hard-coded. Was hoping we could allow them to be changed via the config files but we can refactor that later